doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -727,7 +727,7 @@ bash get_source.sh
 
 You must install a number of software dependencies to create a suitable build environment on your AArch64 Linux system:
 
-- GNU C/C++ compiler (The Docker image uses GCC 7.5)
+- GNU C/C++ compiler 10.3 (The Docker image uses GCC 7.5)
```

Comment:
I'm not sure where the reference to 7.5 came from; I don't think it's currently accurate.

Reply:
I assume it's referring to the alinux docker file
https://github.com/eclipse-openj9/openj9/blob/master/buildenv/docker/aarch64-linux_CC/Dockerfile#L44

Reply:
That makes sense. Thanks.